g11n-ws/modules/md-restful-api/src/main/java/com/vmware/vip/api/rest/APIParamValue.java

```diff
@@ -33,5 +33,8 @@
 	public final static String DEFAULT_TERRITORY = "a boolean value to get the default territory timezone name or not, default is true";
 	public final static String DISPLAY_CITY   = "a flag for displaying all the cities, e.g. true, false";
 	public final static String REGIONS    	  = "a string which represents regions, separated by commas. e.g. US, PT, CN";
+	public final static String USERNAME       = "ldap user name";
+	public final static String PASSWORD       = "ldap user password";
+	public final static String EXPIREDAYS        = "To be able to use the time you are going to apply for Authorization, the unit is one day";
```

Comment:
change the string "To be able to use the time you are going to apply for Authorization, the unit is one day" to "the token's expired days'